openIt/ngsild-payloads/authorization.jsonld

```diff
@@ -61,5 +41,33 @@
         "value": "2024-08-30T11:00:00.000Z"
       }
     }
-  ]
+  ],
+  "forGate": {
+    "type": "Relationship",
+    "object": "urn:ngsi-ld:Gate:123"
+  },
+  "givenBy": {
+    "type": "Relationship",
+    "object": "urn:ngsi-ld:UserProfile:123"
+  },
+  "givenTo": {
+    "type": "Relationship",
+    "object": "urn:ngsi-ld:UserProfile:ABC"
+  },
```

Comment:
du coup sur l'app vous avez un user A qui peux autoriser un user B?

Reply:
Yes, exactly. User A will be giving authorization to User B to open the gate